src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-bar_container">
```

Comment:
The class name 'search-bar_container' should represent the meaning of the content, not the styles or tag names. A more semantic name like 'search-container' might be better.